tamr_toolbox/utils/version.py

```diff
@@ -111,25 +112,31 @@ def does_tamr_version_meet_requirement(
         EnvironmentError: if `raise_error` is True, and the condition is not met
 
     Notes:
+        Patch versions (major.minor.patch) are excluded from the comparison
         If exact_version is True, max_version will be ignored
 
     See Also:
         utils.version.func_requires_tamr_version
     """
 
     error_str = None
+
+    tamr_version_sub = parse(tamr_version).release[:2]
+    min_version_sub = parse(min_version).release[:2]
+
     if exact_version:
-        if not parse(min_version) == parse(tamr_version):
+        if not min_version_sub == tamr_version_sub:
             error_str = f"must be exactly {min_version}."
 
     elif max_version:
-        if parse(min_version) > parse(max_version):
+        max_version_sub = parse(max_version).release[:2]
+        if min_version_sub > max_version_sub:
             raise ValueError("min_version must be smaller than max_version")
 
-        if not parse(min_version) <= parse(tamr_version) <= parse(max_version):
+        if not min_version <= tamr_version <= max_version:
```

Comment:
Should this also be using the "sub" versions?
```suggestion
        if not min_version_sub <= tamr_version_sub <= max_version_sub:
```